tests/object_test.py

```diff
@@ -288,6 +288,17 @@ def __init__(self, value):
         super(ExceptionWithArguments, self).__init__('test')
         self.value = value
 
+class ThingWithExclusion:
+    _jsonpickle_exclude = ["foo"]
```

Comment:
Apologies for the true boring style bikeshed, but please use single quotes like the rest of this file. I'll try and setup `cercis` for formatting the code soon since it enforces my extreme OCD better than `black` does.

You can run `garden fmt` (see `garden.yaml` for details) to deal with other minor stylistic issues if you already have `black` installed.

Reply:
I commented this on the PR implementing cercis, but I don't think ceasing usage of black is necessarily a good idea. See my comments there, but I'm scared that effectively we could be pushing off new contributors with code that doesn't look standard at first glance, and if it doesn't look standard then many people might not devote the effort to contributing to us. I can say that I most likely wouldn't have started contributing to this if it weren't for me seeing the [fade to black](https://github.com/jsonpickle/jsonpickle/commit/56c44b9e541d32a35dff38bfc9ab0986f3821034) commit back in 2020 or so, as that was when I first started looking at jsonpickle.

Reply:
Agree. Please ignore my comment about single-quotes :smile_cat:

Reply:
Made them single quotes anyway to be more consistent. I'm too used to black style :)